### ISSUE TITLE
Feature Request: Improve styling of key/value pairs in the Logs tab

### ISSUE THREAD

**Issue by wojtekn** [Automattic/wp-calypso#96791]
### What

The styling for the key/value that opens under PHP error and Web server logs could be improved to be more consistent with the patterns used elsewhere in the UI.

![Image](https://github.com/user-attachments/assets/da116842-137d-488e-a6fb-344acac7df4b)

### Why

To make UI consistent and improve readability.

### How

Use styling consistent with other parts of UI.

**Comment by Robertght** [comment]
This is how I see it:

![Image](https://github.com/user-attachments/assets/d8c46089-39f7-472b-a47e-1d70d1c7741b)

I'm not sure how we should improve the readability here. @wojtekn could you share more details about this?

**Comment by wojtekn** [comment]
@matt-west I think this finding came from you, could you please advise?

**Comment by matt-west** [comment]
@wojtekn Yep, I’ll follow up with a design. It will probably be in the New Year now as I finish up today.
